doc/man3/BIO_set_callback.pod

```diff
@@ -212,9 +212,22 @@ after.
 The BIO_debug_callback() function is a good example, its source is
 in crypto/bio/bio_cb.c
 
+=head1 RETURN VALUES
+
+BIO_set_callback_ex(), BIO_set_callback() and BIO_set_callback_arg() do not
+return a value.
+
+BIO_get_callback_ex() and BIO_get_callback() return BIO_callback_fn_ex()
+and BIO_callback_fn() respectively.
```

Comment:
This doesn't make sense. BIO_callback_fn_ex and BIO_callback_fn are types?

Reply:
Yes, they are types.

Reply:
I think better would be to say something like "...return the callback function previously set by a call to...". Just saying what the type of the return is, isn't particularly helpful. More useful is saying what it represents.

Reply:
Okay